### PR TITLE
Copyedit release notes

### PR DESCRIPTION
**Pull request by marstr**

### CODE REVIEW

**Comment by marstr** [comment]
@tjprescott mind sanity-checking this for me?

**Comment by sptramer** [comment]
@tjprescott - Please perform the requested sanity check before I do the usual copyedit pass. Should be able to go live tomorrow at the usual time, no problem.

**Comment by tjprescott** [comment]
Looks sane to me.

File: docs-ref-conceptual/release-notes-azure-cli.md [hunk 1]
```diff
@@ -13,6 +13,64 @@ ms.devlang: azurecli
...
+* Initial Release
+
+### Storage
+* BREAKING CHANGE: `storage blob/file/container/share list`- Limit default number of results returned to be 5,000.
+  Use `--num-results *` for original behavior of returning all results.
+* `storage blob/file/container/share list`- log marker for next page to STDERR and expose `--marker` parameter.
+* Add `storage blob service-properties update` command with support for static websites.
+
+### VM
+* 'vm disk/unmanaged-disk' and 'vmss disk': parameter options are more consistent.
```
Comment (sptramer) [inline docs-ref-conceptual/release-notes-azure-cli.md]: Changes like this one need to start having better descriptions:
* Is this a breaking change? (seems like it.)
* What are the old parameter names, and what are the new parameter names?

(cc @tjprescott, @marstr)

**Comment by sptramer** [comment]
@marstr I need to close this PR and reopen it, because you created it off of your `master` branch. This is causing problems with my ability to actually copyedit the release notes. Please resubmit it on a branch.

**Comment by sptramer** [comment]
Opened #1245 which contains the release notes update and the copyedit changes.

**Comment by marstr** [comment]
Just to be clear, @sptramer does #1245 unblock this, or are you still waiting for me to resubmit?

**Comment by sptramer** [comment]
@marstr Unblocked for this release, but keep it in mind for next time

**Pull request closed without merging**